Dockerfile

```diff
@@ -2,16 +2,20 @@
 
 FROM debian:bookworm-slim AS build-bonfhir
 
+ENV URL "http://localhost:8080/fhir"
+ENV VERSION "r4"
+
```

Comment:
This looks great overall, the only thing I would change is to not have a _global_ version, as we discussed earlier each import could have its own version.

Reply:
✅ done